src/protocols/bcp.ts

```diff
@@ -0,0 +1,517 @@
+/**
```

Comment:
Should be in JSON schema since it's a protocol.  Need to merge with existing FDC3 bridging protocol.

Reply:
I'll look into this when I get back from vacation. There are a few reasons to go to JSONSchema (particularly if you want to generate code for other languages to do the same thing)